.github/workflows/test.yml

```diff
@@ -53,14 +81,38 @@ jobs:
       - name: Setup test database
         run: bin/rails db:setup
```

Comment:
I don't think this relates to what you've done, but I'm not sure this db:setup is necessary (in the logs it reports that the database already exists) because the test database has been specified in DATABASE_URL.